src/Mario/Main.purs

```diff
@@ -1,7 +1,9 @@
 module Mario.Main (main) where
 
 import Control.Monad.Eff (Eff())
-import Prelude ((<$>), (<*>), bind, return, unit, Unit())
+import Data.Ring(negate)
```

Comment:
I might be able to make these additional changes tomorrow.

On Aug 27, 2016 7:23 PM, "Michael Ficarra" <notifications@github.com> wrote:

> In src/Mario/Main.purs
> <https://github.com/michaelficarra/purescript-demo-mario/pull/13#discussion_r76524101>
> :
>
> > @@ -1,7 +1,9 @@
> >  module Mario.Main (main) where
> >
> >  import Control.Monad.Eff (Eff())
> > -import Prelude ((<$>), (<*>), bind, return, unit, Unit())
> > +import Data.Ring(negate)
>
> Spacing after this module name.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/michaelficarra/purescript-demo-mario/pull/13/files/2801059c7d5120a33387bb0182ec8e802b284b18#r76524101>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AANyN89ntX4BU7ljK5LbL1pccUtz7Hmvks5qkNTlgaJpZM4Juyce>
> .
>